streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -98,6 +98,11 @@ public void commit() {
         initializeOffsetLimits();
     }
 
+    @Override
+    public void close() {
+        //no-op
```

Comment:
The standby tasks also have state stores, so we should still call `super.closeStateManager()` I think?